catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -30,6 +30,18 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-core</artifactId>
+        </dependency>
```

Comment:
I was unaware, I'll swap the dependencies and make sure all builds well locally 👍

Reply:
And just to clarify: Don't try too hard on the swap, since I'm not sure if the `sun.xml.bind` artifacts are interchangable with the `javax.xml` artifacts.

Reply:
^ Turns out, they're not. Don't worry I didn't try _too_ hard, I swapped them out with their respective latest versions (which were super outdated) and the build was unhappy.

To be fair, these few sun dependencies are used elsewhere in DDF likely for the same reason. So I'm going to leave the sun dependencies as is.